tools/wptserve/wptserve/pipes.py

```diff
@@ -392,6 +392,11 @@ def config_replacement(match):
             value = request.headers
         elif field == "GET":
             value = FirstWrapper(request.GET)
+        elif field == "domains":
```

Comment:
I really don't like this, but I don't think we can reasonably merge `domains` and `not_domains` (given we need to know what's meant to resolve when setting up things like hosts files), and I think we want it like this to resolve #9232.

@jgraham @Ms2ger halp.

Reply:
I think this is probably fine. Maybe write as:

```python
request.server.config.get('not_domains', request.server.config['domains'])
```

(if that's a real `dict`

Reply:
This seems fine to me too.